### PR TITLE
Move vis tool

### PR DESCRIPTION
**Pull request by ceelo777**
- move vis tool to expandable rows
- add logo to indicate row has a visualization
- fix formatting

### CODE REVIEW

File: src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts [hunk 1]
```diff
@@ -317,25 +326,65 @@ export class EventStoreComponent implements OnInit, OnDestroy {
...
-  }
-
-  public findConcurrentEvents(): void {
-    const timelineEventSubs = this.listEventStoreData.map(data => data.eventsList.refresh());
-    forkJoin(timelineEventSubs).subscribe(() => {
-        this.getConcurrentEventsData();
-    });
-  }
+    // grab highcharts data for all events
+    for (let parsedEvent of parsedEvents) {
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts:365]: im curious if there is a better solution for this. adding new properties to an object that doesnt have any type information for it is hard to maintain. Like would it be possible to save the state onto the event store component and pass a way to access the data to the customComponents.

**Review by jeffj6123 — CHANGES_REQUESTED** [review]

File: src/SfxWeb/src/app/modules/concurrent-events-visualization/visualization-tool/visualization-tool.component.scss [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
-    width: 100%;    
+    width: 100%;
     font-size: 5px;
 }
 
 .highcharts-label {
     font-size: 5px;
+}
+
+p {
```
Comment (sidhant012) [inline src/SfxWeb/src/app/modules/concurrent-events-visualization/visualization-tool/visualization-tool.component.scss:11]: if this isn't being used, lets remove it

File: src/SfxWeb/src/app/modules/concurrent-events-visualization/visualization-tool/visualization-tool.component.ts [hunk 3]
```diff
@@ -104,18 +119,20 @@ export class VisualizationToolComponent implements OnInit, OnChanges, AfterViewI
...
                   });
-              } else {
-                config.data.push([`${fontPrefix}${currEvent.eventInstanceId}</p>`, `${fontPrefix}${currEvent.eventInstanceId}</p>`]);
               }
           }  
           levels++;
       }    
       
-      this.options.chart.height = this.minNodeHeight * (levels + 1);      
+      // this.options.chart.height = //this.minNodeHeight * (levels + 1);
```
Comment (sidhant012) [inline src/SfxWeb/src/app/modules/concurrent-events-visualization/visualization-tool/visualization-tool.component.ts:135]: if this isn't used anymore, we can remove it

File: src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts [hunk 4]
```diff
@@ -314,28 +327,70 @@ export class EventStoreComponent implements OnInit, OnDestroy {
...
+        }
+        this.visEventList.push(visEventItem);
 
-    // testing purposes    
-    this.testEvent(parsedEvents);
-  }
+        for (const data of this.listEventStoreData) {                        
+            data.eventsList.collection.forEach(event => {
+                if (Utils.result(event, "raw.eventInstanceId") == Utils.result(parsedEvent, "eventInstanceId")) {   
+                    event['visPresent'] = visPresent;
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts:354]: have this access the property directly and not use indexing this way.

**Review by jeffj6123 — CHANGES_REQUESTED** [review]

**Review by jeffj6123 — APPROVED** [review]

**Pull request merged**